### ISSUE TITLE
Database Repair Link

### ISSUE THREAD

**Issue by bruceperson** [melinsoftware/meos#67]
When attempting to load a database that requires repair, a popup comes up alerting you to that fact. 
It'd be great if that pop-up also had abutton that can take you directly into database repair of that requested database, rather than having to hunt about finding the Repair button, and re-selecting the target database, particularly as this situation is most likely to encountered in the field.

**Comment by erikmelin** [comment]
I think it is very rare that the database actually needs a repair; actually I have never seen it in a real event. And I am not sure the database can even detect if that is the case.

**Comment by bruceperson** [comment]
This happened at an event. The plugpack was knocked out of the laptop with the MeOS database, and eventually the battery went flat, causing an immediate shutdown. I suspect there is a flag that isn't unset when a database isn't shut down properly that indicates the database should be checked before use again. A bit like fsck runs on Linux when there has been an improper shutdown. We did not know there was a database check button in a Menu in MeOS (never having had to use it), so we had to put the rest of the competitors into another empty event, and then attempt to merge the two events later.
